packages/page-claims/src/Attest.tsx

```diff
@@ -65,7 +66,7 @@ function Attest ({ accountId, className, ethereumAddress, statementKind }: Props
             icon='send'
             isPrimary
             label={t('Attest')}
-            params={[statementKind]} // Replace with actual
+            params={[getStatement(statementKind)]}
```

Comment:
Probably want to memo/state this param since it gets called each render. 

(As it stands the arrays we pass everywhere is horrible anyway since it will change when re-rendered, but have not gotten to that cleanup, but at least we probably don't want to make a "the call" here)